snuba/perf.py

```diff
@@ -90,12 +92,24 @@ def write():
 
     time_start = time.time()
     if profile_process:
-        cProfile.runctx('process()', globals(), locals(), sort='cumulative')
+        filename = tempfile.NamedTemporaryFile(
+            prefix=os.path.basename(events_file) + '.process.',
+            suffix='.pstats',
+            delete=False,
+        ).name
+        cProfile.runctx('process()', globals(), locals(), filename=filename)
```

Comment:
you are not passing the sort argument anymore. Is that intended?

Reply:
Yeah, the sort can be applied after the fact with the pstats CLI, so there's no reason to pass it here.

Reply:
ok